src/ert/analysis/_es_update.py

```diff
@@ -244,31 +230,8 @@ def _create_temporary_parameter_storage(
     param_group: str,
 ) -> TempStorage:
     temp_storage = TempStorage()
-    t_genkw = 0.0
-    t_surface = 0.0
-    t_field = 0.0
-    _logger.debug("_create_temporary_parameter_storage() - start")
```

Comment:
This proved a bit useful earlier today, so perhaps add some kind of info logging? Could be on `save_parameters` or inside `_save_temp_storage_to_disk`